dbt_platform_helper/utils/aws.py

```diff
@@ -349,6 +350,46 @@ def get_postgres_connection_data_updated_with_master_secret(session, parameter_n
     return parameter_data
 
 
+def get_redis_supported_versions(elasticache_client=None):
```

Comment:
Reads a little better...
```suggestion
def get_supported_redis_versions(elasticache_client=None):
```